bitcoin/src/merkle_tree/block.rs

```diff
@@ -732,7 +732,7 @@ mod tests {
     #[cfg(feature = "rand-std")]
     impl PartialMerkleTree {
         /// Flip one bit in one of the hashes - this should break the authentication
-        fn damage(&mut self, rng: &mut ThreadRng) {
+        fn damage(&mut self, rng: &mut prelude::ThreadRng) {
```

Comment:
This flat out looks wrong, the `prelude::` bit does not give any additional information to the call site, so is just noise.

Reply:
Agreed, and removed